relays/bin-substrate/src/chains/rococo_parachains_to_bridge_hub_wococo.rs

```diff
@@ -76,3 +76,10 @@ impl CliBridgeBase for BridgeHubRococoToBridgeHubWococoCliBridge {
 	type Source = relay_bridge_hub_rococo_client::BridgeHubRococo;
 	type Target = relay_bridge_hub_wococo_client::BridgeHubWococo;
 }
+
+impl MessagesCliBridge for BridgeHubRococoToBridgeHubWococoCliBridge {
+	const ESTIMATE_MESSAGE_FEE_METHOD: &'static str =
```

Comment:
(not related to the PR) It should be removed in #1638 (sorry - there are still some traces of the old code).